src/routes/Board/Board.tsx

```diff
@@ -54,6 +54,11 @@ export const Board = () => {
 
   const currentUserIsModerator = state.participants?.self.role === "OWNER" || state.participants?.self.role === "MODERATOR";
 
+  if (state.participants?.self.banned) {
+    window.location.reload();
+    return <LoadingScreen />; // fallback
```

Comment:
I think a modified RejectionPage with a "You have been banned from this session" description would be more fitting

Reply:
Is there anything like this designed already?